module/Activity/view/activity/admin/list.phtml

```diff
@@ -17,6 +17,7 @@
         <th><?= $this->translate('Organ') ?></th>
         <th><?= $this->translate('Submitter') ?></th>
         <th><?= $this->translate('Update status') ?></th>
+        <th><?= $this->translate('Approved') ?></th>
```

Comment:
There is no reason to display the approval status in an admin list, where this value is already shown by the table in which it appears (note that this table is not used for old activities)